src/ray/gcs/redis_context.h

```diff
@@ -173,6 +167,21 @@ class RedisContext {
   instrumented_io_context &io_service() { return io_service_; }
 
  private:
+  /// Run an arbitrary Redis command synchronously.
+  ///
+  /// \param args The vector of command args to pass to Redis.
+  /// \return CallbackReply(The reply from redis).
+  std::unique_ptr<CallbackReply> RunArgvSync(const std::vector<std::string> &args);
+
+  void ValidateRedisDB();
```

Comment:
Do you mind adding some doc?

Something I'm interested:
- What do you validate?
- What if validation fail? Do you throw exception, exit, or return sth?

Reply:
The original function does not have comments too, so I think this can be fixed in a separate PR.